.circleci/test-deploy.yml

```diff
@@ -165,7 +165,20 @@ jobs:
       - matlab/run-tests:
           test-results-junit: test-results/matlab/filterdtagresult.xml
           select-by-tag: FILTERED
-
+      - matlab/run-tests:
+          select-by-name: $(circleci tests glob 'tests/**/*.m' | circleci tests split | awk -F'[\\\\/.]' '{print $(NF-1) "/*"}')
+          source-folder: src
+      - matlab/run-tests:
+          select-by-name: "mytest/* filtertest/*"
+          source-folder: src
+      - matlab/run-tests:
+          select-by-name: $(circleci tests glob 'tests/**/*.m' | awk -F'[\\\\/.]' '{print $(NF-1)}' | circleci tests split --split-by=timings | awk '{print $0 "/*"}')
+          test-results-junit: test-results/matlab/results.xml   
+          source-folder: src
+      - matlab/run-tests:
+          select-by-name: $(circleci tests glob 'tests/**/*.m' | circleci tests split --split-by=filesize | awk -F'[\\\\/.]' '{print $(NF-1) "/*"}')
+          test-results-junit: test-results/matlab/results.xml   
+          source-folder: src
```

Comment:
Need some kind of corresponding verification that these invocations did what was expected.

Reply:
I have added a verification step for this feature